hr_timesheet_hours_billed/tests/test_hours_billed.py

```diff
@@ -0,0 +1,117 @@
+# Copyright 2023-nowdays Cetmix OU (https://cetmix.com)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl)
+from odoo.tests import tagged
+
+from odoo.addons.sale_timesheet.tests.common import TestCommonSaleTimesheet
+
+
+@tagged("-at_install", "post_install")
+class TestCommonHourBilled(TestCommonSaleTimesheet):
+    # @classmethod
+    def setUp(self):
```

Comment:
Please move this to setUpClass and disable tracking unless you need it for functional purposes
```

    @classmethod
    def setUpClass(cls):
        super().setUpClass()
        cls.env = cls.env(context=dict(cls.env.context, tracking_disable=True))
```